#!/usr/bin/python3
def raise_exception():
    # Calling raise_exception() will immediately raise a TypeError
    raise TypeError('Exception raised')
